### DISCUSSION TITLE
v1.0 Release time ?

### DISCUSSION THREAD

**Discussion by dapeng17951** [nuejs/nue#111] [Announcements]
v1.0 Release time ?

**Comment by tipiirai** [comment]
I'm planning to release a new minor version every two months or so. The next one will be v0.2.0 and the ETA is next month. v1.0.0 is a major one. It needs to be production-ready so it has to be properly tested in many different real-world sites and scenarios. Hard to estimate the time. Definitely not this year.

**Comment by ShiMeiWo** [comment]
The progress of the implementation can be seen in this figure:
https://nuejs.org/why/#why-nue

Let's look forward to it.

**Comment by wdcodecn** [comment]
when Nue CSS release ?

**Comment by tipiirai** [comment]
Nuekit was released yesterday: https://nuejs.org/blog/nuekit-010/

Going to release Nuemark before Nue CSS, because **content first**
